docs/developers-guide/driver-changelog.md

```diff
@@ -4,6 +4,13 @@ title: Driver interface changelog
 
 # Driver Interface Changelog
 
+## Metabase 0.49.9
+
+- Another driver feature has been added: `upload-with-auto-pk`. It only affects drivers that support `uploads`, and
```

Comment:
I think it's worth mentioning that this feature will also cause this column to be stripped from uploads that already contain it.

Reply:
I'm not sure I understand, can you be more specific with the exact wording you were looking for? If the feature is supported, all upload tables will contain the `_mb_row_id`. And that's already documented [here](https://www.metabase.com/docs/latest/databases/uploads#primary-key-auto-generation). Maybe I should just link to this section of the docs?

Reply:
It's pretty fringe, but I was thinking of something like this:

> When this feature is false, all upload tables will have the metabase id columns omitted, even if they were explicitly present in the input file, e.g. from exporting a previous model.

Reply:
Ah right well that's not how this works. The `_mb_row_id` column sticks around if the feature is false. 
See `load-from-csv-auto-pk-column-non-supporting-test`:
https://github.com/metabase/metabase/blob/0289f9f23cf1e9d8afb489237bab848c08fa93d6/test/metabase/upload_test.clj#L836-L852

Reply:
Ah neat, way prefer this behavior. Tripped up with all the use of `without-auto-pk-columns`.

To make sure I see the full picture - we'll regenerate the column values if the feature is true, or keep the old values if it's false? Perhaps still worth a call-out.

Reply:
> we'll regenerate the column values if the feature is true, or keep the old values if it's false?

Yes.

> Perhaps still worth a call-out.

Good point, this isn't explained very clearly. Perhaps this is better to be documented above the declaration of the feature itself in `metabase.driver`, rather than in the changelog where it's more likely to be forgotten. I'll add that in the v50 PR.
https://github.com/metabase/metabase/blob/0289f9f23cf1e9d8afb489237bab848c08fa93d6/src/metabase/driver.clj#L590-L592

Reply:
Perfect